doc/api/modules.md

```diff
@@ -132,6 +132,14 @@ variable. Since the module lookups using `node_modules` folders are all
 relative, and based on the real path of the files making the calls to
 `require()`, the packages themselves can be anywhere.
 
+## Addenda: The .mjs extension
+
+Currently it is not possible to require files that have the `.mjs` extension,
```

Comment:
```suggestion
Currently it is not possible to require files that have the `.mjs` extension.
```

Reply:
I ended up relanding this change... I think , rather than . is appropriate